user/user-guide.md

```diff
@@ -196,9 +196,7 @@ The `DictionaryService` provides meta-information about the metadata model.
 It allows to fetch information about registered types, aspect and properties.
 
 ### Viewing JavaDoc
-Full JavaDoc documentation of the Alfred API Java interface is available in the JavaDoc. You can
-view the JavaDoc by browsing to `/alfresco/s/apix/javadocs/index.html` on your Alfresco host that
-has Alfred API installed.
+Full JavaDoc documentation of the Alfred API Java interface is available on this site at [https://docs.xenit.eu/alfred-api/stable-user/javadoc/](https://docs.xenit.eu/alfred-api/stable-user/javadoc/).
```

Comment:
I thought Javadoc would only be available via Maven Central?

Reply:
I thought you wanted to host it unpacked on the site? Mavencentral just has the archive. We can just refer to that if you'd like.